internal/command/jsonfunction/parameter.go

```diff
@@ -0,0 +1,42 @@
+package jsonfunction
+
+import (
+	"github.com/zclconf/go-cty/cty/function"
+)
+
+// parameter represents a parameter to a function.
+type parameter struct {
+	// Name is an optional name for the argument.
+	Name string `json:"name,omitempty"`
+
+	// Description is an optional human-readable description
+	// of the argument
+	Description string `json:"description,omitempty"`
+
+	// IsNullable is true if null is acceptable value for the argument
+	IsNullable bool `json:"is_nullable"`
```

Comment:
Nitpick: Have you considered making this `omitempty` too, given how many `false`s there are, as that could potentially save some bytes in the total output size?

Reply:
Good catch! I've added the flag and updated the JSON output.